catalog/spatial/registry/registry-schema-bindings/src/main/java/org/codice/ddf/registry/schemabindings/helper/InternationalStringTypeHelper.java

```diff
@@ -85,13 +91,37 @@ public InternationalStringType create(String internationalizeThis) {
     public void setLocale(Locale locale) {
         this.locale = locale;
     }
+    
+    public void setNearestMatch(boolean findNearestMatch) {
+        this.findNearestMatch = findNearestMatch;        
+    }
 
     private Optional<String> getLocalizedString(List<LocalizedStringType> localizedStrings) {
         Optional<String> optionalLocalString = localizedStrings.stream()
                 .filter(localizedString -> locale.toLanguageTag()
                         .equals(localizedString.getLang()))
                 .findFirst()
                 .map(LocalizedStringType::getValue);
+        
+        //If an exact match has not been found then look at the base language e.g. if en-GB 
+        //has not been found then try to find the first one in the list of localizedStrings 
+        //that has the same language e.g. en-US. The behaviour of this will vary depending
+        //on the order that the localized strings are loaded
+        if (!optionalLocalString.isPresent() && findNearestMatch)
```

Comment:
Agreed, a fallback would be useful and more deterministic but that would mean that something like Locale.China (zh-cn) would fallback to en-US for example. Should both nearest match and default be implemented for flexibility? Or just put in a default as people should load up the languages they want and that will mean its clear when a localization is missing.